test/proposals.unit.tests.js

```diff
@@ -0,0 +1,329 @@
+var BridgeTestable= artifacts.require("./BridgeTestable");
+var ProposalsContract = artifacts.require("./ProposalsContract");
+var PreserveBalancesOnTransferToken = artifacts.require("./PreserveBalancesOnTransferToken");
+const time = require('./helpers/time');
+
+require('chai')
+	.use(require('chai-as-promised'))
+	.use(require('chai-bignumber')(web3.BigNumber))
+	.should();
+
+contract('ProposalsContract unit tests', (accounts) => {
+	var creator = accounts[0];
+	var u1 = accounts[1];
+	var u2 = accounts[2];
+	var u3 = accounts[3];
+	var u4 = accounts[4];
+	var u5 = accounts[5];
+	var outsider = accounts[6];
+
+	var bridgeTestable;
+	var proposalsContract;
+	var preserveBalancesOnTransferToken;
+
+	var EXIT_STAKE = 0;
```

Comment:
this variable (and some of the others) is not used. Please remove all the code which is not used.

Please consider running your code through eslint as well. I understand the code will still have warnings as our eslint setup it is not finetuned yet, but it will still catch a lot of things anyway. Thanks